src/todos/todos.module.ts

```diff
@@ -0,0 +1,13 @@
+import { MikroOrmModule } from '@mikro-orm/nestjs';
+import { Module } from '@nestjs/common';
+import { Todo } from 'src/common/entities/todo.entity';
+import { TodosController } from './todos.controller';
+import { TodosService } from './todos.service';
+import { TodoRepository } from './todos.repository';
+
+@Module({
+  imports: [MikroOrmModule.forFeature([Todo])],
+  controllers: [TodosController],
+  providers: [TodosService, TodoRepository],
```

Comment:
```suggestion
  providers: [TodosService],
```

You don't need to provide repository if you add the repository to the entity. Importing just the entity should be fine